test/unit/delete_publisher.test.ts

```diff
@@ -0,0 +1,60 @@
+import { connect } from "../../src"
+import { expect } from "chai"
+import { Rabbit } from "../support/rabbit"
+import { randomUUID } from "crypto"
+import { expectToThrowAsync, username, password } from "../support/util"
+
+describe("DeletePublisher command", () => {
+  const rabbit = new Rabbit(username, password)
+  const testStreamName = "test-stream"
+  let publisherRef: string
+
+  beforeEach(async () => {
+    publisherRef = randomUUID()
+    await rabbit.createStream(testStreamName)
+  })
+
+  afterEach(() => rabbit.deleteStream(testStreamName))
+
+  it("can delete a publisher", async () => {
+    const connection = await connect({
+      hostname: "localhost",
+      port: 5552,
+      username,
+      password,
+      vhost: "/",
+      frameMax: 0,
+      heartbeat: 0,
+    })
+    const publisher = await connection.declarePublisher({ stream: testStreamName, publisherRef })
+    await publisher.send(Buffer.from(`test${randomUUID()}`))
+
+    const publisherId = publisher.getPublisherId()
+
+    const deletePublisher = await connection.deletePublisher(Number(publisherId))
+    expect(deletePublisher).eql(true)
+
+    await connection.close()
```

Comment:
When write a test please keep the AAA style:http://wiki.c2.com/?ArrangeActAssert 
```suggestion
    const publisher = await connection.declarePublisher({ stream: testStreamName, publisherRef })
    await publisher.send(Buffer.from(`test${randomUUID()}`))

    const publisherId = publisher.getPublisherId()

    const deletePublisher = await connection.deletePublisher(Number(publisherId))
    expect(deletePublisher).eql(true)
    await connection.close()
```

Reply:
Resloved in this commit https://github.com/coders51/rabbitmq-stream-js-client/commit/7cbc2d08f70627cf6a7270380a5f4e9019919369